lighthouse-core/report/html/report-styles.css

```diff
@@ -993,9 +994,17 @@
 .lh-warnings--toplevel {
   --item-margin: calc(var(--header-line-height) / 4);
   color: var(--report-text-color-secondary);
-  margin: var(--category-padding);
-  padding: var(--category-padding);
+  margin-left: auto;
+  margin-right: auto;
+  width: 50%;
+  background-color: var(--color-amber);
```

Comment:
feels like we might want a bit more padding with the new background style 🤷‍♂

Reply:
done and updated images